frontend/apps/app/components/SessionDetailPage/components/Output/components/Artifact/utils/__tests__/formatArtifactToMarkdown.test.ts

```diff
@@ -0,0 +1,1151 @@
+                        {
+                          executed_at: '2024-06-03T11:00:00Z',
+                          success: false,
+                          result_summary:
+                            'ERROR: update or delete on table "airplanes" violates foreign key constraint "fk_flights_airplane"',
+                        },
+                      ],
+                    },
+                  ],
+                },
+              ],
+            },
+            {
+              name: 'Performance',
+              description: [
+                'The system must be able to search 1000 flight records within 3 seconds and support concurrent access from 50 users.',
+              ],
+              type: 'non_functional',
+            },
+            {
+              name: 'Security',
+              description: [
+                'Pilot personal information must be encrypted for storage and access logs must be recorded. Database access must be restricted to authenticated users only.',
+              ],
+              type: 'non_functional',
+            },
+          ],
+        },
+      }
+
+      const result = formatArtifactToMarkdown(artifact)
+      expect(result).toMatchInlineSnapshot(`
+        "# Requirements Document
+
+        This document outlines system requirements and their associated data manipulation language (DML) operations.
+
+        ---
+
+        ## 📋 Business Requirements
+
+        Define database requirements for consistently managing airline-owned aircraft, operated flights, and pilots (captains and first officers) involved in operations, while maintaining scheduled and actual times for flights. This enables consistent tracking of aircraft used and personnel assigned for each flight.
+
+        ## 🔧 Functional Requirements
+
+        ### 1. Aircraft Management
+
+        - The company owns multiple aircraft, each with an aircraft number, model, and seating capacity. The aircraft number must be unique, and seating capacity must be an integer greater than 0.
+
+
+        **Use Cases:**
+
+        #### 1.1. Aircraft Registration and Update
+
+        The operations manager opens the aircraft management screen, selects "New Registration", and enters the aircraft number, model, and seating capacity. Upon clicking the register button, the system verifies required field entry, confirms aircraft number uniqueness, checks that seating capacity is a positive number, then records it in the aircraft registry.
+
+        ##### **INSERT** - New aircraft registration test including normal cases and edge cases (capacity 1/0, model names with special characters and non-ASCII).
+
+        \`\`\`sql
+        BEGIN;
+        INSERT INTO airplanes (airplane_number, model, capacity) VALUES
+          ('JA100A', 'ATR42-600', 48),
+          ('JA200B', 'Bombardier CRJ900', 90),
+          ('JA330C', 'Flying Car Alpha', 5);
+        COMMIT;
+        \`\`\`
+
+        **Execution History:**
+
+        ✅ **06/01/2024, 08:00:00 AM**
```

Comment:
While ``June 1, 2024, 8:05 AM`` might feel more natural in English-speaking regions than ``06/01/2024, 08:00:00 AM,`` I think it’s perfectly fine to keep it as it is for now.

Reply:
Thanks, I'd like to follow up and make some adjustments here 👍🏻